credsweeper/file_handler/data_content_provider.py

```diff
@@ -46,6 +45,11 @@ def data(self, data: bytes) -> None:
         """data setter"""
         self.__data = data
 
+    def __is_structure(self) -> bool:
```

Comment:
__has_structure ? __is_structured_data ? __format_recognized ?

Reply:
too complicated question for the private member name :)